seedfarmer/cli_groups/_bootstrap_group.py

```diff
@@ -97,20 +97,12 @@ def bootstrap_toolchain(
         project = _load_project()
     _logger.debug("Bootstrapping a Toolchain account for Project %s", project)
     bootstrap_toolchain_account(
-        project_name="exampleproj",
-        principalARNs=trusted_principal,
-        permissionsBoundaryARN=permission_boundary,
```

Comment:
i knew there was something i didn't like about those variable names, but i couldn't figure out what it was...